### PR TITLE
Fix Overlay bug + Footer padding adjustments 

### PR DESCRIPTION
**Pull request by chrismekim**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -18,7 +20,10 @@ const Footer = ({ hidden }) => (
     </div>
     <div className="flex justify-end items-end">
       <Link
-        className="small link decoration-none"
+        className={cx('small link decoration-none', {
```
Comment (chrismekim) [inline src/components/Footer.js:23]: Added so the color of the "Working with us" link changes.

File: src/views/MainView.js [hunk 2]
```diff
@@ -55,6 +55,9 @@ class MainView extends PureComponent {
 
   componentWillUnmount() {
     clearInterval(timerID);
+
+    const html = document.documentElement;
```
Comment (limsohee1002) [inline src/views/MainView.js:59]: Should we use getter here?

Comment (chrismekim) [reply src/views/MainView.js:59]: Haha I asked Hugh this too, and he said that we know that the document will always be there so there is no need to use a getter.

Comment (hhff) [reply src/views/MainView.js:59]: Ya, this code is safe. But usually YES you should use a GETTER

**Pull request merged**